### ISSUE TITLE
command to toggle indent rainbows?

### ISSUE THREAD

**Issue by jots** [oderwat/vscode-indent-rainbow#11]
Is it feasible to hook up a keystroke to toggle the rainbows on and off?

**Comment by oderwat** [comment]
Whats the reasoning behind it: Aesthetics or performance?

**Comment by obahareth** [comment]
I need this as well, for aesthetic reasons. Indent rainbow is very helpful, but I don't need it all of the time. Sometimes all the colors can get distracting.

**Comment by Ovyerus** [comment]
I agree. I only really need this when I'm cleaning up loose indents or working with something like Python, and having a command to easily toggle this on and off would be a major quality of life improvement IMO.

**Comment by fmaiabatista** [comment]
I'd also benefit from this ability. I often want to turn it off to clear the view.
